src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -137,8 +137,8 @@ public bool TryWriteProperties(PropertiesWriter writer, out IEnumerable<ProjectD
             var rootModuleFiles = PutFilesToRightModuleOrRoot(validProjects, rootProjectBaseDir);
```

Comment:
It would be super nice to fix the coding style in this file (operator positions, method parameters, etc.) and to also make some methods static, however I am afraid that the coverage will drop a lot in that case. 

If @andrei-epure-sonarsource also agrees to do this cosmetics, he might be able to save us and merge the PR even if it has low coverage...

Reply:
I would suggest to do a separate PR for large cosmetic changes

it would make the review harder and also confuse around what's not covered - changed functionality vs refactoring

Reply:
I agree with @andrei-epure-sonarsource.
The PR is already starting to be pretty big IMO.